staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -767,7 +783,11 @@ func (c *Cacher) GetList(ctx context.Context, key string, opts storage.ListOptio
 		// minimal resource version, simply forward the request to storage.
 		return c.storage.GetList(ctx, key, opts, listObj)
 	}
-	if listRV == 0 && utilfeature.DefaultFeatureGate.Enabled(features.ConsistentListFromCache) {
+	requestWatchProgressSupported, err := etcdfeature.DefaultFeatureSupportChecker.Supports(storage.RequestWatchProgress)
```

Comment:
```suggestion
	requestWatchProgressSupported, err := c.storage.Supports(storage.RequestWatchProgress)
```

Reply:
We already have `Supports` on cache that do the same i will change it with that. 

Lets say we have a problem with parsing the version this will run `Status` every time. Is it ok?

Reply:
Yes, this is ok as it's part of handling a request List request. Such request can hit etcd every time.